coops-ui/src/views/Dashboard.vue

```diff
@@ -12,40 +12,80 @@
               <header>
                 <h2 class="mb-3">To Do <span class="text-muted">({{todoCount}})</span></h2>
               </header>
-              <todo-list @todo-count="todoCount = $event"
-                         @has-blocker-filing="hasBlockerFiling = $event"
-                         @todo-filings="todoListFilings = $event"
-                         :inProcessFiling="inProcessFiling" />
+              <todo-list
+                @todo-count="todoCount = $event"
+                @has-blocker-filing="hasBlockerFiling = $event"
+                @todo-filings="todoListFilings = $event"
+                :inProcessFiling="inProcessFiling"
+                :coaPending="coaPending"
+              />
             </section>
             <section>
               <header>
                 <h2 class="mb-3">Recent Filing History <span class="text-muted">({{filedCount}})</span></h2>
               </header>
-              <filing-history-list @filed-count="filedCount = $event"
-                                   @filings-list="historyFilings = $event" />
+              <filing-history-list
+                @filed-count="filedCount = $event"
+                @filings-list="historyFilings = $event"
+              />
             </section>
           </div>
 
           <aside class="dashboard-content__aside">
             <section>
               <header class="aside-header mb-3">
                 <h2>Office Addresses</h2>
-                <v-btn text small color="primary" id="btn-standalone-addresses" :disabled="hasBlockerFiling"
-                      @click.native.stop="goToStandaloneAddresses()">
+                <v-scale-transition>
+                  <v-tooltip
+                    top
+                    content-class="pending-tooltip"
+                  >
+                    <template v-slot:activator="{ on }">
+                      <v-chip
+                        small
+                        label
+                        color="yellow"
+                        text-color="black"
+                        v-show="coaPending"
+                        v-on="on"
+                      >Pending
+                      </v-chip>
+                    </template>
+                    <span>
+                      The updated office addresses will be legally effective on {{ effectiveDate }}, 12:01 AM(PST).
```

Comment:
I don't think we can say this is "Standard" time (since we might currently be in Daylight time). In other places we just say "Pacific Time". Check with Scott or your designer on this.

Reply:
Ok great, thanks for pointing that out, i will update for consistency!